sentry/src/main/java/io/sentry/SentryReplayOptions.java

```diff
@@ -139,6 +139,16 @@ public enum SentryReplayQuality {
    */
   private boolean debug = false;
 
+  /**
+   * The screenshot strategy to use for capturing screenshots during replay recording. Defaults to
+   * {@link ScreenshotStrategyType#PIXEL_COPY}. If set to {@link ScreenshotStrategyType#CANVAS}, the
+   * SDK will use the Canvas API to capture screenshots, which will always mask all Texts and
+   * Bitmaps drawn on the screen, causing {@link #addMaskViewClass} and {@link #addUnmaskViewClass}
+   * to be ignored.
+   */
+  @ApiStatus.Experimental
+  private @NotNull ScreenshotStrategyType screenshotStrategy = ScreenshotStrategyType.PIXEL_COPY;
```

Comment:
Wondering if we should also expose it via the Manifest? I think we already do have something for enums e.g. for profiling